splot/_viz_mpl.py

```diff
@@ -0,0 +1,224 @@
+"""
+Lightweight visualizations for pysal using Matplotlib and Geopandas
+
+TODO
+add Examples
+change function input naming to be in line with bokeh functionality
+"""
+__author__ = ("Stefanie Lumnitz <stefanie.lumitz@gmail.com>")
+
+import matplotlib.pyplot as plt
+import geopandas as gpd
+import pysal as ps
+import splot.plot
+
+from matplotlib import patches, colors
+
+from ._viz_utils import mask_local_auto
+
+
+def lisa_cluster(moran_loc, df, p=0.05, figsize=None, ax=None,
+                 legend=True, legend_kwds=None):
+    """
+    Create a LISA Cluster map
+    
+    Parameters
+    ----------
+    moran_loc : esda.moran.Moran_Local instance
+        Values of Moran's Local Autocorrelation Statistic
+    df : geopandas dataframe instance, optional
```

Comment:
currently it is not "optional" because it is a positional argument in the function signature, so you actually *need* to pass one

Reply:
ok, this was changed df is not an "optional' input